dev-games/simgear/simgear-2018.2.2.recipe

```diff
@@ -0,0 +1,95 @@
+SUMMARY="A Simulator Construction Toolkit"
+DESCRIPTION="Set of open-source libraries designed as building blocks \
+for quickly assembling 3D simulations, games, and visualization applications."
+HOMEPAGE="https://home.flightgear.org/"
+COPYRIGHT="2002-2018 Curtis Olsen and others"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/flightgear/simgear-$portVersion.tar.bz2"
+CHECKSUM_SHA256="f61576bc36aae36f350154749df1cee396763604c06b8a71c4b50452d9151ce5"
+PATCHES="simgear-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	simgear$secondaryArchSuffix = $portVersion
+	lib:libsimgear$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libosg$secondaryArchSuffix
+	lib:libplibnet$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	simgear$secondaryArchSuffix = $portVersion
+	devel:libsimgear$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	simgear$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libboost_system$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libgl$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libopenal$secondaryArchSuffix
+	devel:libosg$secondaryArchSuffix
+	devel:libplibfnt$secondaryArchSuffix
+	devel:libplibjs$secondaryArchSuffix
+	devel:libplibnet$secondaryArchSuffix
+	devel:libplibpsl$secondaryArchSuffix
+	devel:libplibpu$secondaryArchSuffix
+	devel:libplibpuaux$secondaryArchSuffix
+	devel:libplibpw$secondaryArchSuffix
+	devel:libplibsg$secondaryArchSuffix
+	devel:libplibsl$secondaryArchSuffix
+	devel:libplibsm$secondaryArchSuffix
+	devel:libplibssg$secondaryArchSuffix
+	devel:libplibssgaux$secondaryArchSuffix
+	devel:libplibul$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir -p build && cd build
+	cmake .. $cmakeDirArgs \
+		-DSIMGEAR_SHARED:BOOL=ON \
+		-DENABLE_TESTS:BOOL=OFF
+	make $jobArgs
+}
+
+INSTALL()
+{
+	cd build
+	make -C install
+	rm $libDir/*.la
+	mkdir -p `dirname $includeDir` $libDir
+	mv $prefix/include $includeDir
+
+	if [ -n "$secondaryArchSuffix" ]; then
+		mv $prefix/lib/libSimGear* $prefix/lib/pkgconfig $libDir/
```

Comment:
You can probably simplify `INSTALL` if you pass a few extra parameters to cmake in `BUILD`:
```diff
 	cmake .. $cmakeDirArgs \
+		-DCMAKE_INSTALL_PREFIX:PATH="$prefix" \
+		-DCMAKE_INSTALL_INCLUDEDIR:PATH="$includeDir" \
+		-DCMAKE_INSTALL_LIBDIR:PATH="$libDir" \
 		-DSIMGEAR_SHARED:BOOL=ON \
 		-DENABLE_TESTS:BOOL=OFF
```
(I did not test this yet.)